resume.html

```diff
@@ -1,10 +1,67 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Personal Website - Resume</title>
+    <title>Joyce Kwon</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
+
   <body>
-    
+  	<h1>joyce kwon</h1>
+
+  	<h2>RESUME</h2>
+
+	  	<h3>Education</h3>
+
+		  	<h4>New York University - New York, NY</h4>
+			  	<p>Bachelor of Music in Instrumental Performance – Cello</p>
+			  	<p>Minor: Digital Art and Design</p>
+
+		  	<h4>Study Abroad at NYU Prague</h4>
+			  	<p>New York University, Prague, Czech Republic</p>
+
+  		<h3>Relevant Experience</h3>
+
+  			<h4>Vice President of Design</h4>
+				<p>Communications Club, New York, NY</p>
+					<ul>
+					<li>Design club event posters and other promotional materials</li>
+					<li>Rebrand logos, website and apparel</li>
+					<li>Design graphics for social media newsletter and other aesthetic contents</li>
+					<li>Lead company site visits</li>
+					<li>Document visual and motion contents at club events</li>
+					</ul>
+
+			<h4>Social Media Marketing and UX Design Intern</h4>
+				<p>MAKEUS Mobile, Seoul, South Korea</p>
+					<ul>
+					<li>Conducted a marketing research on social media and successful marketing strategies</li>
+					<li>Formulated effective guerrilla marketing ideas</li>
+					<li>Executed a promotional event that involved 8,000 participants and users</li>
+					<li>Operated an intra-company film contest with the budget scale of 10,000 US dollars</li>
+					<li>Constructed business models for efficient user acquisition</li>
+					</ul>
+
+			<h4>Global Marketing Intern</h4>
+				<p>StyleShare Inc., Seoul, South Korea</p>
```

Comment:
You can leave elements on the same line as the h4 above. They are sibling elements and usually sibling are kept at the same tab indentation.